python/lsst/jointcal/jointcal.py

```diff
@@ -756,7 +756,7 @@ def _load_data(self, inputSourceTableVisit, inputVisitSummary, associations, joi
             The filter bands of each input dataset.
         """
         oldWcsList = []
-        bands = []
+        filters = []
```

Comment:
Here and below, I think that `filters` is actually `filterLabels`?  If so, I think you should be explicit in the name because `filter` is such an overloaded term at the moment.  Instead of these are physical filters this should be `physicalFilters`, but I think they're labels.

Reply:
Argh, github won't let me add a comment above that the docstring is incorrect, it isn't returning `bands` it's returning something else (`filterLabels` or `physicalFilters`).

Reply:
@kfindeisen suggested that I leave these as "filter/filters" because that is what the RFC process will eventually settle on everywhere else (once the deprecation period is over), and there was no deprecation danger here. `filter` in the pipelines code will mean `filterLabel` pretty much everywhere.

See the "Transition" section here: https://community.lsst.org/t/migrating-from-afw-image-filter-to-filterlabel/4591